ValidationLibrary/FixableRuleBase.cs

```diff
@@ -115,10 +115,10 @@ protected async Task<IReadOnlyList<RepositoryContent>> GetContents(IGitHubClient
                  * Octokit.NotFoundException. This same thing would probably be throw if the whole repository
                  * was missing, but we don't care for that case (no point to validate if repository doesn't exist.)
                  */
```

Comment:
Please move this back to `ValidationLibrary.Rules`. `FixableRuleBase` is an implementation detail that should be close to rules.

In startup check just do `!c.IsAbstract` to avoid adding it to DI.

Reply:
I thought it might go there because the IValidationRule is there and just to leave the rules in the rules folder as you guessed but I have moved it back.